src/index.css

```diff
@@ -77,7 +77,7 @@
 @layer components {
   .modal-position {
     position: fixed;
-    top: 45%;
+    top: 55%;
```

Comment:
```suggestion
    top: 240;
```
로 바꾸면 모달이 화면 크기에 관계없이 메시지 아래에 위치합니다.

Reply:
반영했습니다